README.md

```diff
@@ -49,6 +50,14 @@ In order to use the generator, there are 2 possible endpoints that can be used.
 - DELETE: Sending a DELETE request on the endpoint `{Base URL}/test-data/company/{companyNumber}` will delete the test company. There is a required parameter that is Authcode which needs to be included in the request body to be allowed to delete the test company. An usage example looks like this: `{"auth_code":"222222"}`
 - Health Check: Sending a GET request on the endpoint `{Base URL}/test-data/healthcheck` will return a status code and an empty response body.
 
+#### Creating test users
+- POST: Sending a POST request to create users with the associated roles `{Base URL}/test-data/users` will generate a new test user. The request body can include `UserSpec` parameter to customise the generated user.
+    - `password`: The password of the user.
+    - `roles`: The roles of the user along with `permissions`. This is optional and defaults to an empty list.
```

Comment:
It does not look like `permissions` defaults to an empty list but throws an error if not provided?